ddtrace/appsec/__init__.py

```diff
@@ -3,9 +3,11 @@
 
 def load_appsec():
     """Lazily load the appsec module listeners."""
-    from ddtrace.appsec._asm_request_context import listen
+    from ddtrace.appsec._asm_request_context import asm_listen
+    from ddtrace.appsec._iast._iast_request_context import iast_listen
 
     global _APPSEC_TO_BE_LOADED
     if _APPSEC_TO_BE_LOADED:
-        listen()
+        asm_listen()
+        iast_listen()
```

Comment:
Shouldn't we keep the load_appsec function only for threats and put iast in its own `load_iast` function?